### DISCUSSION TITLE
Any news on a new release?

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#914] [General]
Just wanted to follow up on any news regarding a new release. I have been personally using Beta 4 as it is the most stable version.

Thanks!!

**Comment by CaBuffalo** [comment]
Andreas has been busy on other projects. but ! He said he will come back to
work on AG whenever he's available. :)

Cheers.

On Fri, Feb 25, 2022 at 11:50 AM jaacMusic ***@***.***> wrote:

> Just wanted to follow up on any news regarding a new release. I have been
> personally using Beta 4 as it is the most stable version.
>
> Thanks!!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/914>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AVTO7F54I3MP53VPLU47MOXANCNFSM5PLEL7FQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Pete3030** [comment]
That's good news! I am also using latest Beta, it's pretty stable here so far.. Would be great to see a final version anytime soon..;)
